examples/octet_stream/text-client.cpp

```diff
@@ -103,7 +103,6 @@ int caf_main(caf::actor_system& sys, const config& cfg) {
   sys.println("*** server is running, enter an empty line (or CTRL+D) to stop");
   auto line = std::string{};
   while (std::getline(std::cin, line)) {
-    sys.println("line: {}", line);
```

Comment:
This has been removed to resolve CI failures. This code only prints what the user has added as input. Sometimes, an output can come between these lines and the robot test fails to find desired output in file.